dashboard/test/serializers/api/v1/pd/enrollment_flat_attendance_serializer_test.rb

```diff
@@ -178,7 +178,13 @@ class Api::V1::Pd::EnrollmentFlatAttendanceSerializerTest < ::ActionController::
   end
 
   test 'extract school and teacher info when they are empty' do
-    enrollment = build :pd_enrollment, school_info: (build :school_info_us)
+    enrollment = build(
```

Comment:
Why are we using `create` on line 168 but `built` here?

Reply:
Because it this case the existing code cares about the `school_info` object being built but not created, whereas the other one doesn't have any such restrictions.